src/main/java/de/craftlancer/clcapture/CLCapture.java

```diff
@@ -167,43 +161,27 @@ public void saveDefaultConfig() {
         if (!pointsFile.exists())
             try {
                 Files.createFile(pointsFile.toPath());
-            }
-            catch (IOException e) {
+            } catch (IOException e) {
                 Bukkit.getLogger().log(Level.WARNING, "Error while creating points.yml", e);
             }
     }
     
-    @EventHandler(ignoreCancelled = true, priority = EventPriority.LOWEST)
-    public void onSignCreation(SignChangeEvent event) {
-        if (!event.getLine(0).equalsIgnoreCase(SIGN_HEADER))
-            return;
-        
-        if (!event.getPlayer().hasPermission(ADMIN_PERMISSION)) {
-            event.setCancelled(true);
-            return;
-        }
-        
-        String type = event.getLine(1);
-        String name = event.getLine(2);
-        String id = event.getLine(3);
-        
-        Block chestLocation = event.getBlock().getRelative(BlockFace.DOWN);
-        
+    public void pointAdd(Player sender, Location chestLocation, String type, String name, String id) {
         if (!types.containsKey(type))
-            event.getPlayer().sendMessage("This capture point type does not exist!");
+            sender.sendMessage("This capture point type does not exist!");
         else if (name.isEmpty() || id.isEmpty())
-            event.getPlayer().sendMessage("You must specify a name and an ID!");
+            sender.sendMessage("You must specify a name and an ID!");
         else if (points.stream().map(CapturePoint::getName).anyMatch(a -> a.equals(name)))
-            event.getPlayer().sendMessage("A capture point with this name already exists!");
+            sender.sendMessage("A capture point with this name already exists!");
         else if (points.stream().map(CapturePoint::getId).anyMatch(a -> a.equals(name)))
-            event.getPlayer().sendMessage("A capture point with this id already exists!");
-        else if (chestLocation.getType() != Material.CHEST && chestLocation.getType() != Material.TRAPPED_CHEST)
-            event.getPlayer().sendMessage("The sign must be placed directly above a Chest!");
+            sender.sendMessage("A capture point with this id already exists!");
+        else if (chestLocation.getBlock().getType() != Material.CHEST && chestLocation.getBlock().getType() != Material.TRAPPED_CHEST)
+            sender.sendMessage("You must be looking at a chest!");
         else {
-            CapturePoint point = new CapturePoint(this, name, id, types.get(type), chestLocation, event.getBlock());
+            CapturePoint point = new CapturePoint(this, name, id, types.get(type), chestLocation.getBlock());
             points.add(point);
             savePoints(true);
-            event.getPlayer().sendMessage("CapPoint successfully created!");
+            sender.sendMessage("CapPoint successfully created!");
         }
     }

```

Comment:
Change to 
```
public boolean addPoint(CapturePoint point) {
    boolean success = points.add(point);
    savePoints(true);
    return success;
}
```
and move the entire validation and messaging logic to PointAddCommand#execute.